src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+  public void draw() {
+        System.out.println(
```

Comment:
to do such refactor everywhere
```suggestion
    @Override
    public void draw() {
        System.out.println(
```